code/modules/mob/living/carbon/monkey/combat.dm

```diff
@@ -272,7 +272,11 @@
 				if(Adjacent(target) && isturf(target.loc))	// if right next to perp
 
 					// check if target has a weapon
-					var/obj/item/weapon/W = locate(/obj/item/weapon) in target.held_items
+					var/obj/item/weapon/W
+					for(var/obj/item/weapon/I in target.held_items)
+						if(!istype(I, /obj/item/weapon/twohanded/offhand)
```

Comment:
It should check for the [ABSTRACT](https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/flags.dm#L19) flag instead